android/settings.gradle

```diff
@@ -1,3 +1,5 @@
 rootProject.name = 'treemapper'
+include ':react-native-localize'
+project(':react-native-localize').projectDir = new File(rootProject.projectDir, '../node_modules/react-native-localize/android')
```

Comment:
Since react-native > 0.60 we use auto-linking of module - isn't that possible for `react-native-localize`? If yes, please use "react-native unlink react-native-localize" to get rid of this stuff again.

Reply:
ok. i will do that. Thanks

Reply:
I did it for you as I also wanted to merge the latest changes about the new .env module into all current PRs.